assignment-2/power.md

```diff
@@ -51,21 +53,29 @@ According to Robert Kennedy [8] a Google employee that worked in is free time in
 
 It seems that this culture of fear resulted that on vital reports were not reacted on the right way. For example on April 4, 2013 a 15-page document by McKinsey & Co. warns among others the CMS administrator that there was insufficient end-to-end testing and advises that a limited initial launch of the website would be ideal. This reports surfaced only half November 2013. One and a half month after the lauch. The advise in this report was not followed, because there were only two weeks of end-to-end testing before the launch took place for all the users and there was no limited initial lauch. Also other reports and warnings were discarded like that the website as of September 30 could handle only about 1100 users at a time, even though officials have said it should have been able to accomodate perhaps as many as 60,000 users. It seems that people were afraid to be fired or were afraid that the Republicans would use such reports to try to delay or cancel the whole project and with that the whole Obamacare bill.
 
-
 Robert Kennedy [8] also stated that he met with an employee of one of the site contractors who was in charge of software “security.” When he asked her which penetration threats she had prepared for, she responded that she didn’t think about such things: she was in charge of “risk assessment.”. In contractor terms, that meant the risk of government contract nonfulfillment. She saw that her job was to ensure that a software audit confirmed that the software was “secure” as defined in the contract. So the overall security objective was not to make sure that the system was difficult to hack, but that the contractual terms were met.
 
-######Intervention
-The question that you could ask is after reading the analysis is why did not someone intervine? This is not a question that I can answer, but I can speculate about this with the information we have.
+####Intervention
+The question that you could ask is after reading the previous part is: why does someone does not intervine and ends the project or replaces contractors? This is not a question that I can answer, but I can speculate about this with the information we have.
```

Comment:
Double `is`: `The question that you could ask after reading the previous part is`.